modules/ossm-upgrade-known-issues.adoc

```diff
@@ -7,7 +7,7 @@
 
 Known issues that may affect your upgrade include:
 
-* {SMProductName} does not support the use of `EnvoyFilter` configuration except where explicitly documented. If you are using Envoy Filters, and the configuration generated by Istio has changed due to the new version of Envoy introduced by upgrading your `ServiceMeshControlPlane`, that has the potential to break any `EnvoyFilter` you may have implemented.
+* {SMProductName} does not support the use of `EnvoyFilter` configuration except where explicitly documented. This is due to tight coupling with the underlying Envoy APIs, meaning that backward compatibility cannot be maintained. If you are using Envoy Filters, and the configuration generated by Istio has changed due to the new version of Envoy introduced by upgrading your `ServiceMeshControlPlane`, that has the potential to break any `EnvoyFilter` you may have implemented.
```

Comment:
```suggestion
* {SMProductName} does not support the use of `EnvoyFilter` configuration except where explicitly documented. This is due to tight coupling with the underlying Envoy APIs, meaning that backward compatibility cannot be maintained. If you are using Envoy Filters, and the configuration generated by Istio has changed due to the latest version of Envoy introduced by upgrading your `ServiceMeshControlPlane`, that has the potential to break any `EnvoyFilter` you may have implemented.
```

Reply:
This suggestion is per the entry for 'new' in the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage